Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -81,15 +81,15 @@ public void Ctor_NullLogger_Throws()
         [TestMethod]
         public void LogWSOnError()
         {
-            this.downloader.Pages["http://myhost:222/api/qualityprofiles/search?projectKey=foo+bar"] = "trash";
+            this.downloader.Pages["http://myhost:222/api/qualityprofiles/search?project=foo+bar"] = "trash";
```

Comment:
Ask discussed offline, this will be breaking change for users of old versions of SQ so it should be handled separately.